pep-0295.txt

```diff
@@ -5,118 +5,127 @@ Last-Modified: $Date$
 Author: yozh@mx1.ru (Stepan Koltsov)
 Status: Rejected
 Type: Standards Track
+Content-Type: text/x-rst
 Created: 22-Jul-2002
 Python-Version: 3.0
 Post-History:
 
+
 Abstract
+========
 
-    This PEP describes an interpretation of multiline string constants
-    for Python.  It suggests stripping spaces after newlines and
-    stripping a newline if it is first character after an opening
-    quotation.
+This PEP describes an interpretation of multiline string constants
+for Python.  It suggests stripping spaces after newlines and
+stripping a newline if it is first character after an opening
+quotation.
 
 
 Rationale
+=========
+
+This PEP proposes an interpretation of multiline string constants
+in Python.  Currently, the value of string constant is all the
+text between quotations, maybe with escape sequences substituted,
+e.g.::
 
-    This PEP proposes an interpretation of multiline string constants
-    in Python.  Currently, the value of string constant is all the
-    text between quotations, maybe with escape sequences substituted,
-    e.g.:
-
-        def f():
-            """
-            la-la-la
-            limona, banana
-            """
-        
-        def g():
-            return "This is \
-            string"
-        
-        print repr(f.__doc__)
-        print repr(g())
-    
-    prints:
-    
-        '\n\tla-la-la\n\tlimona, banana\n\t'
-        'This is \tstring'
-    
-    This PEP suggest two things
-
-	- ignore the first character after opening quotation, if it is
-	  newline
-	- second: ignore in string constants all spaces and tabs up to
-	  first non-whitespace character, but no more than current
-	  indentation.
-
-    After applying this, previous program will print:
-    
-        'la-la-la\nlimona, banana\n'
-        'This is string'
-    
-    To get this result, previous programs could be rewritten for
-    current Python as (note, this gives the same result with new
-    strings meaning):
-    
-        def f():
-            """\
+    def f():
+        """
         la-la-la
         limona, banana
         """
-        
-        def g():
-            "This is \
+
+    def g():
+        return "This is \
         string"
-    
-    Or stripping can be done with library routines at runtime (as
-    pydoc does), but this decreases program readability.
+
+    print repr(f.__doc__)
+    print repr(g())
+
+prints::
+
+    '\n\tla-la-la\n\tlimona, banana\n\t'
+    'This is \tstring'
+
+This PEP suggest two things:
+
+- ignore the first character after opening quotation, if it is
+  newline
+
+- second: ignore in string constants all spaces and tabs up to
```

Comment:
While we are here we can delete "second:". "This PEP suggest two things:" is already implies that here are two items.